internal/httputil/client.go

```diff
@@ -105,7 +104,7 @@ func Do(ctx context.Context, method, endpoint, userAgent string, headers map[str
 			endpoint = u.String()
 		}
 	default:
-		return fmt.Errorf(http.StatusText(http.StatusBadRequest))
+		return errors.New(http.StatusText(http.StatusBadRequest))
```

Comment:
Could we put in any lint changes in a separate PR?

Reply:
Sure, this pr will fail checks though